aries-site/src/examples/templates/feedback/ModalTaskFlowExample/TaskFlowFeedback.js

```diff
@@ -0,0 +1,117 @@
+import { useCallback, useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  StarRating,
+  Text,
+  TextArea,
+} from 'grommet';
+import { ButtonGroup } from '../../ButtonGroup';
+
+// These credentials will be supplied when working with unified
+// customer experience team and should likely be stored in environment varibles.
+const API_URL = process.env.FEEDBACK_API_URL;
+const API_TOKEN = process.env.FEEDBACK_API_TOKEN;
+
+export const TaskFlowFeedback = ({ onClose }) => {
+  const router = useRouter();
+  const url = `${router.route}`;
+  // The properties within the surveyValues object will be specific to your
+  // product and instance within the product. These will be specified
+  // when working with the unified customer experience team.
+  const surveyValues = {
+    fullURL: url,
+    QID1: undefined,
+    QID2_TEXT: '',
+  };
+  const [formValue, setFormValue] = useState(surveyValues);
+  const [thankyou, setThankyou] = useState(false);
+
+  const onSubmit = useCallback(({ value }) => {
+    const data = {
+      values: value,
+    };
+
+    const submitResponse = () =>
+      fetch(API_URL, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+          'X-API-TOKEN': API_TOKEN,
+        },
+        body: JSON.stringify(data),
+      })
+        .then(response => {
+          // Response handling here
+          // update state/routing as approiate based on response.json
+        })
+        .catch(error => {
+          // Error handling here
+        });
+
+    //  preventing api call for demo purposes
+    //  submitResponse();
+  }, []);
+
+  return (
+    <Form
+      kind="survey"
```

Comment:
It would be nice to highlight in the guidance that getting the proper styling comes from applying `kind="survey"` to the underlying form. Feel like that's important.

Reply:
done